components/Document/DocumentViewer.tsx

```diff
@@ -89,6 +85,8 @@ const DocumentViewer = ({
   setReferenceItemDatum,
   referenceItemDatum,
   documentViewerClass,
+  withControls = true,
+  allowAnnotations = true,
```

Comment:
Adding new config options because zoom controls and annotations don't make sense on question type